pkr/ext/__init__.py

```diff
@@ -148,11 +148,18 @@ def list_all(cls):
             module = importer.find_module(package_name).load_module(package_name)
             extensions[package_name] = cls._get_extension_class(module)
         # Load from pkr_extensions entrypoints (and TO BE DEPRECATED extensions group)
-        for entry in entry_points().get("pkr_extensions", ()) + entry_points().get(
-            "extensions", ()
-        ):
+        eps = entry_points()
+        if hasattr(eps, "select"):
+            extension_eps = list(eps.select(group="pkr_extensions")) + list(
+                eps.select(group="extensions")
```

Comment:
I am not sure this works with all pythons (3.7 if I remember well)

Reply:
It does:
3.6 and 3.7 need the pypi package importlib_metadata (already in our requirements for python < 3.8), then the value returned from entry_points() does have the "select" method;
3.8 and 3.9 load the built-in "entry_points()" where there is no "select" method, but "get".

I wrote these lines to get rid of the "deprecation warning". Funnily though it warns me that I'd need to use "select"... maybe the future as seen in 3.6 and 3.7 warnings was not as it turned out to be.